src/test/scala/nl.knaw.dans.easy.bagstore/ItemIdSpec.scala

```diff
@@ -121,4 +160,10 @@ class ItemIdSpec extends TestSupportFixture {
       case Success(b) => b shouldBe bagId
     }
   }
+
+  private def expectValidationToFailOnUuidLength(nonsenseUuid: String)  = {
+       fromString(nonsenseUuid) should matchPattern {
```

Comment:
Could you run the formatter you installed today on your changes?

Reply:
done